backend/src/main/java/com/crm/api/v1/CarController.java

```diff
@@ -7,11 +7,7 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.data.domain.Page;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
+import org.springframework.web.bind.annotation.*;
```

Comment:
Nie potrzebujemy wszystkiego. Zmień w ustawieniach intellij tak aby optymalizator importów nie wstawiał Ci *

Reply:
No ale tego nie ma. Przy szczegółowych importach jest `-`, a przy tym z `*` jest `+`

Reply:
Mateuszowi chodziło o to, że jak jest * to zaciąga cały pakiet, a my potrzebujemy tylko kilku klas. W opcjach Intellij można to zmienić, domyślnie jest chyba, że dla 5 i więcej importów zmienia się na gwiazdkę. Ja ustawiłem sobie 999 i teraz już mi tego IntelliJ nie zmienia, tylko importuje te klasy, które faktycznie używam.

Reply:
Tak to jest, że trzeba umieć czytać ze zrozumieniem. Poprawione.